salt/personalised-covers/init.sls

```diff
@@ -64,6 +64,14 @@ personalised-covers-logs:
         - require:
             - file: personalised-covers-cache
 
+personalised-covers-data:
+    file.recurse:
+      - name: /srv/personalised-covers-data/formats
+      - source: salt://personalised-covers/data/formats
+      - user: {{ pillar.elife.webserver.username }}
```

Comment:
files only need to be read and the `elife` user `pillar.elife.deploy_user` is part of the group. So this is correct